index.html

```diff
@@ -26,9 +24,21 @@
 </head>
 
 <body>
-   
-    <main id="content container">
+    <!-- This is the navigation bar -->
+    <nav class="hide-on-med-and-down">
+        <ul>
+            <li><a href="#home">Home</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#tracks">Tracks</a></li>
+            <li><a href="#schedule">Schedule</a></li>
+            <li><a href="#faq">FAQ</a></li>
+            <li><a href="#sponsors">Sponsors</a></li>
+            <!-- <li><a class="apply" target="_blank" href="https://tinyurl.com/hacknc2021application">Register</a></li> -->
```

Comment:
Can you remove this commented out code, as well as the Volunteer and Mentor Sign up link?